server/routes/filmsRoutes.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose');
+const Film = mongoose.model('films');
+const User = mongoose.model('users');
```

Comment:
Do you have future plans to use this? It is currently not being used in the file?

I would suggest removing it to keep the code clean and free of redundancy